docs/synthetics/ssl-certificate-monitoring.md

```diff
@@ -39,10 +40,32 @@ The Browser monitor loads the website in a real Google Chrome browser and perfor
 
 Sematext Synthetics checks the certificate expiry every day and alerts you via the monitor's configured [alert notification hooks](../alerts/alert-notifications.md) multiple times before it expires. We make sure you're reminded about the expiry multiple times.
 
+You can set the certificate expiry alert condition within Conditions tab for both HTTP and Browser monitors
+
+![Certificate expiry BM](../images/synthetics/cert-expiry2.png)
+
+Or from SSL Monitoring tab for HTTP Monitors
+
+![Relaxed SSL certificate check](../images/synthetics/ssl-check-type.png)
+
 The monitor performs the expiry check for all the certificates in the chains - leaf, intermediate, or root certificates. The **Monitor Overview** page displays the expiry time of the recently expiring certificate.
 
 ![Certificate expiry](../images/synthetics/cert-expiry.png)
 
+## Certificate Authority
+
+Sematext Synthetics checks the certificate authority for both Root or Intermediate level and alerts you via the monitor's configured [alert notification hooks](../alerts/alert-notifications.md) if the conditions are not met.
+
+The alert condition is supported for both HTTP and Browser monitors and can be set from Conditions tab
+
+![CA check](../images/synthetics/cert-authority.png)
+
+
+Using a Browser monitor with a user journey script that visits multiple pages will perform the checks on the SSL of the first page only.
```

Comment:
We do capitalize user journey script elsewhere?
will perform the checks on the SSL ... ==> will perform SSL certificate checks for the first page only

Reply:
Checked other pages, capitalized U & J